src/app/page.tsx

```diff
@@ -4,11 +4,22 @@ import QuoteBox from "@/components/home/QuoteBox";
 import quotes from "@/data/quote-data";
 import EHTitle from "@/components/events/EventHighlightsTitle";
 import MissionStatement from "@/components/home/MissionStatement";
-
+import Image from "next/image";
 const Home = () => {
   return (
-    <div className="bg-csa-tan-100 pt-[12.2vh] md:pt-[14.4vh]">
-      <MissionStatement />
+    <div className="bg-csa-tan-100">
+      <div className="grid grid-flow-col grid-rows-3 gap-4">
+        <Image
+          src="/home/dragon.png"
```

Comment:
```suggestion
          src="/home/dragon.png"
```
Import it at the top as so -- 
`Import Dragon from "@/public/home/dragon.png"`
then your src here will just be src={Dragon}